libs/sr/page-team/src/BuildsDisplay.tsx

```
@@ -166,52 +152,24 @@ function BuildBase({
+      <LightConeCardCompact bgt="light" lightConeId={lightConeId} />
       {allRelicSlotKeys.map((slot) => (
-        <GridItem
-          key={`${slot}_${relics[slot]?.id}`}
-          slot={slot}
-          relic={relics[slot]}
+        <RelicCardCompact
+          bgt="light"
+          relicId={relicIds[slot]}
+          slotKey={slot}
+          showLocation
         />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)